net/sysrepo/Makefile

```diff
@@ -141,8 +137,8 @@ ifeq ($(CONFIG_SYSREPO_PYTHON),y)
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(PKG_BUILD_DIR)/swig/libSysrepo-cpp.so* $(1)/usr/lib
 	$(INSTALL_DIR) $(1)$(PYTHON_PKG_DIR)
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/swig/python2/libsysrepoPython2.py $(1)$(PYTHON_PKG_DIR)
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/swig/python2/_libsysrepoPython2.so $(1)$(PYTHON_PKG_DIR)
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/swig/python3/libsysrepoPython3.py $(1)$(PYTHON_PKG_DIR)
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/swig/python3/_libsysrepoPython3.so $(1)$(PYTHON_PKG_DIR)
```

Comment:
Should these references to `PYTHON_PKG_DIR` be for `PYTHON3_PKG_DIR` instead?

Reply:
Also, are there packages that depend on these Python bindings? Do they need Python 2 bindings or can they work with Python 3?

I'm wondering if we should (and if it is possible to) offer both Python 2 and 3 bindings, keeping the Python 2 bindings around until the mass removal in 2020.

Reply:
I don’t think anything uses these. Feel free to open a PR fixing up some of the loose ends.

Reply:
See #10554 - I decided against building Python 2 bindings because it would have been difficult to build both Python 2 and 3 bindings (but not totally impossible), but also as you said, nothing here uses it.

In fact, if you wanted these bindings before, you had to manually build the package with the config option set. If someone really needs Python 2 bindings, they can edit the Makefiles and build it themselves.